target/linux/realtek/dts-5.10/rtl8380_d-link_dgs-1210-10mp-f.dts

```diff
@@ -8,6 +8,42 @@
 	compatible = "d-link,dgs-1210-10mp-f", "realtek,rtl8382-soc", "realtek,rtl838x-soc";
 
 	model = "D-Link DGS-1210-10MP F";
+
+	// i2c of the left SFP cage: port 9
```

Comment:
Use `/* ... */` for comments (like below).

Reply:
Fixed on both files.